src/ui/search-xref.js

```diff
@@ -29,8 +29,8 @@ const button = ui.addCommand(l10n.title, show, "Ctrl+Shift+Alt+x", "📚");
 function show() {
   const onLoad = e => e.target.classList.add("ready");
   const xrefSearchUI = html`
-    <iframe id="xref-ui" src="${XREF_URL}" onload=${onLoad}></iframe>
-    <a href="${XREF_URL}" target="_blank">Open Search UI in a new tab</a>
+    <iframe class="respec-iframe" src="${URL}" onload=${onLoad}></iframe>
```

Comment:
```suggestion
    <iframe class="respec-iframe" src="${URL}" onload="${onLoad}"></iframe>
```

Reply:
We're using hyperHTML so we don't need to use quotes, but all cool.